Assets/MixedRealityToolkit/_Core/Sharing/StateControl/Logging/AppStateLogger.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Pixie.AppSystems.TimeSync;
+using Pixie.Core;
+using Pixie.Initialization;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using Newtonsoft.Json;
```

Comment:
Is netwonsoft a dependency already existing for the mrtk? If not, are there instructions for how to pick up this dependency/should we start taking this on as a dependency?

Reply:
sounds good

Reply:
```
using Newtonsoft.Json;
```
Noooooooo. Should be using the Unity JSON NOT Newtonsoft.  This should be avoided at ALL costs